packages/tools/viewer/src/viewerAnnotationElement.ts

```diff
@@ -109,11 +112,22 @@ export class HTML3DAnnotationElement extends LitElement {
         this._viewerAttachment = null;
 
         this._internals.states?.add("invalid");
+
+        this._updateAnnotation = null;
     }
 
     /** @internal */
     // eslint-disable-next-line @typescript-eslint/naming-convention
     protected override render() {
         return html` <slot></slot> `;
     }
+
+    /** @internal */
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    protected override update(changedProperties: PropertyValues<this>): void {

```

Comment:
why don't we stick to the naming convention here?

Reply:
This is an override from the LitElement base class provided by the Lit library.